unittests/AST/ASTImporterTest.cpp

```diff
@@ -1066,5 +1087,49 @@ TEST_F(Fixture,
   Check(To);
 }
 
+TEST_F(Fixture, IDNSOrdinary) {
+  Decl *From, *To;
+  std::tie(From, To) = getImportedDecl(
+    R"(
+    void declToImport() {}
+    )",Lang_CXX, "", Lang_CXX);
+
+  MatchVerifier<Decl> Verifier;
+  auto Matcher = functionDecl();
+  ASSERT_TRUE(Verifier.match(From, Matcher));
+  EXPECT_TRUE(Verifier.match(To, Matcher));
+  EXPECT_EQ(From->getIdentifierNamespace(), To->getIdentifierNamespace());
+}
+
+// Matcher class to retrieve the last matched node under a given AST.
+template <typename NodeType> class Matcher : public MatchFinder::MatchCallback {
```

Comment:
I would name this according to its role... like LastMatcher instead of a too general name.